### ISSUE TITLE
[UI] Notification center not being filtered out based on Event type.

### ISSUE THREAD

**Issue by Jougan-0** [meshery/meshery#11458]
### Current Behavior

In Notification Center the selectors for Info, Error, Success, Warnings don't filter out the events based on the Event Type.
### Desired Behavior

It should filter the events based on the Event type.
### Screenshots/Logs

![Screenshot from 2024-07-25 05-24-45](https://github.com/user-attachments/assets/de6b55f5-067d-4d19-8747-3f52b1436c1a)

### Environment

- Browser: Chrome Safari Firefox
- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

**Comment by Jougan-0** [comment]
@MUzairS15 @aabidsofi19 //

**Comment by leecalcote** [comment]
I experience the same issue. I wonder if this isn't due to slow performance in the remote provider's event retrieval.

**Comment by jerensl** [comment]
@leecalcote I see through the redux dev tools, it does not update the state as well but also triggers it and sometimes it goes undefined, can I work with this issue?

**Comment by jerensl** [comment]
@leecalcote Hi Lee, 

Let me elaborate more on my approach, I think it's supposed to use [redux-thunk](https://redux-toolkit.js.org/usage/usage-with-typescript#createasyncthunk) for asynchronous data fetching. When dealing with the nested object and array I also recommended adding an extra layer with Immer to make it work with immutable data. So my work will be a primary refactor redux event slice. What do you think about my approach?

**Comment by aabidsofi19** [comment]
@jerensl  thanks for looking into . We do already use rtk for the async fetching and that happens inside the loadEvents  thunk.

**Comment by aabidsofi19** [comment]
@jerensl  the pagination params were recently also changed and be causing the issue

**Comment by jerensl** [comment]
> @jerensl api params ( pagination , filters ) were recently also changed and might be causing the issue

Yup, the problem is the pagination indexing which starts from 0 on the server but from the web, it starts from 1. The web fetch data from `/api/v2/events?severity=%5B"informational"%5D&status=unread&page=1&sort=created_at&order=desc&pagesize=15` which are empty, the page params should be 0 I guess? but I still don't know which part code affected the pagination page
